modules/core/spec/01_concepts.md

```diff
@@ -4,28 +4,6 @@ order: 1
 
 # Concepts
```

Comment:
This document still has information that needs to be consolidated, but the rest can be done in a followup (which is why this pr doesn't close the original issue). Opening this pr as an example so another contributor can help with finishing off this for core IBC and applying this to ics20

Reply:
Can you please write a comment in the ticket with what the leftover work will be?

Reply:
The leftover work is all the docs remaining in the `spec` directory. Everything must be deleted or moved. Should I be more detailed?

Reply:
I see that you have created some new issues, so I think that's fine. Thanks!